exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,9 @@
 // Start by creating a variable `message`
-
+var messageTypes = "My name is ";
+var name = "victoria ";
+var messageType = "and my name is ";
+var nameLength = name.length;
+var many = " characters long.";
+var message = messageTypes + name + messageType + name.length + many;
```

Comment:
I would be mindful of your naming of variables. Try to make them as descriptive as possible so you don't confuse yourself or someone else looking at the code later down the line. I would personally not have assigned the the parts of your sentence as individual variables also. I would have just had the name and maybe name length as variables.

e.g var message = "My name is " + name + " and my name is" + nameLength + "characters long.";